tests/modules/nf-core/bwa/aln/main.nf

```diff
@@ -34,6 +34,6 @@ workflow test_bwa_aln_paired_end {
     ]
     fasta = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
 
-    BWA_INDEX ( fasta )
+    BWA_INDEX ( [ [:], fasta ] )
```

Comment:
no nice meta map, like you did in the other PR?

Reply:
I was lazy and copy pasted from BWA_MEM2 😊

Reply:
There, fixed it for you!